stock_outgoing_shipment_report/models/stock_outgoing_shipment_report.py

```diff
@@ -55,6 +56,9 @@ class StockOutgoingShipmentReport(models.Model):
     customer_delivery_note = fields.Char("Customer Delivery Note")
     client_order_ref = fields.Char("Customer Reference")
     memo = fields.Char("Memo")
+    date_created = fields.Date(
+        "Created Date", default=fields.Date.context_today, store=True
+    )
```

Comment:
👍 Just adjust the label.

Reply:
Edit from "Created Date" to "Created Date(for date designation)".